contracts/registry/ServiceAuthorizationV3.sol

```diff
@@ -171,6 +175,19 @@ contract ServiceAuthorizationV3
      function _setupComponentServiceAuthorization()
           internal
      {
+          IAccess.FunctionInfo[] storage functions;
```

Comment:
I don't think these all open authorizations here (also valid for changes further down in the contrac
t) are a good idea as those call should be limited to the calling service and not fully public.

Reply:
this functions are used by components (not service to service) and must be disabled in case of release lock. This is done through restricted modifier. While `_getAndVerifyCallingComponent()` and similar functions are limiting this functions to components of specific version and type.

Reply:
service functions called by components:
 need to be granted to the public role. 
 caller validation needs to be done inside of those functions. 

service functions callec by other services:
 must not be granted to the public role
 caller validation needs to be done throuhg service roles